scripts/run_integration.sh

```diff
@@ -230,12 +253,14 @@ function expand_codescape_config() {
     "mips64")
       MIPS_FLAGS="-EB -mips64r6 -mabi=64"
       FLAVOUR="mips-r6-hard"
+      #MIPS_FLAGS="-EB -mips64r2 -mabi=64"
```

Comment:
same here and below, why do you need to keep these comments?

Reply:
same here maybe we could have a flag to decide which version to use so user can easily reuse this script to target the correct CPU by just changing one line, so while we only enable one flavor it's easy for the community to adapt.

Or again we should/could have both `mips-r6`/`mips-r2` and a `mips` alias to target the last one aka `mips-r6` ?